client/src/components/layout/MovieList.js

```diff
@@ -25,13 +25,15 @@ const MovieList = (props) => {
 
     const moviesListArray = movies.map((movie)=> {
         return (
+            
             <MovieTile movie={movie} key={movie.id}/>
+           
```

Comment:
hmmm ideally this is either a one line return, or at least there arent these two empty lines.